documentation/info/collect.py

```diff
@@ -66,6 +66,7 @@ def execute(context):
 
     # Income
     df_income_municipality = context.stage("data.income.municipality")
+    df_income_municipality = df_income_municipality[(df_income_municipality["attribute"] == "all") & (df_income_municipality["modality"] == "all")]
```

Comment:
I think `modality` comes from `modalité`, I saw this a couple of times, but I think it is a false friend in French. I think in English `value` is mostly used for that.

Reply:
Valentin is asking Pierre-Olivier what would be the correct word to use. We could indeed use "value", it is a very generic and broadly used keyword but the attribute/value pair seems logic.

Reply:
Ah yes, if it is coming out of the library this is not easy to change here. Let's not block merging this PR because of that, except you think this can be fixed quickly. :)

Reply:
I renamed the column after reading from library and before running optimisation. It works. We can see later if we should change it in the library too.